pages/index.vue

```diff
@@ -138,9 +138,8 @@
       <v-spacer size="triple" />
     </div>
     <!-- news river -->
-    <loading-icon v-if="!riverLoaded" />
     <section
-      v-if="river"
+      v-if="riverSection && !$fetchState.pending"
```

Comment:
I guess the con of using fetchState.pending is that you can't load things incrementally, it has to be all at once